mbed_lstools/lstools_base.py

```diff
@@ -267,6 +267,12 @@ def __init__(self, **kwargs):
         "9012": "SEEED_TINY_BLE",
         "9900": "NRF51_MICROBIT",
         "C002": "VK_RZ_A1H",
+        "C030": "C030-U201",
+        "C031": "C030-N211",
+        "C032": "C030-R404M",
+        "C033": "C030-R410M",
+        "C034": "C030-S200",
+        "C035": "C030-R3121",
```

Comment:
I just pasted in what @screamerbg gave me by e-mail:

BOARD NAME: C030-U201
BOARD ID: C030
BOARD URL: https://developer.mbed.org/platforms/ublox-C030-U201/
BOARD NAME: C030-N211
BOARD ID: C031
BOARD URL: https://developer.mbed.org/platforms/ublox-C030-N211/
BOARD NAME: C030-R404M
BOARD ID: C032
BOARD URL: https://developer.mbed.org/platforms/ublox-C030-R404M/ 
BOARD NAME: C030-R410M
BOARD ID: C033
BOARD URL: https://developer.mbed.org/platforms/ublox-C030-R410M/ 
BOARD NAME: C030-S200
BOARD ID: C034
BOARD URL: https://developer.mbed.org/platforms/ublox-C030-S200/ 
BOARD NAME: C030- R3121
BOARD ID: C035
BOARD URL: https://developer.mbed.org/platforms/ublox-C030-R3121/ 

Rob